it/uk/gov/hmrc/incorporatedentityidentificationfrontend/views/CheckYourAnswersViewTests.scala

```diff
@@ -118,11 +120,15 @@ trait CheckYourAnswersViewTests {
                               (result: => WSResponse,
                                companyNumberStub: => StubMapping,
                                authStub: => StubMapping,
-                               insertJourneyConfig: => Future[WriteResult]): Unit = {
+                               insertJourneyConfig: => Future[WriteResult],
+                               auditStub: => StubMapping,
+                               retrieveCtutrStub: => StubMapping): Unit = {
```

Comment:
These need refactoring as we have done on some of the other services - we'll have to get a ticket on that.